manual/cops_sorbet.md

```diff
@@ -293,12 +293,13 @@ You can configure the placeholders used by changing the following options:
 
 * `ParameterTypePlaceholder`: placeholders used for parameter types (default: 'T.untyped')
 * `ReturnTypePlaceholder`: placeholders used for return types (default: 'T.untyped')
+* `Style`: signature style to enforce - 'sig' for sig blocks, 'rbs' for RBS comments, 'both' to allow either (default: 'sig')
 
 ### Configurable attributes
 
 Name | Default value | Configurable values
 --- | --- | ---
-AllowRBS | `false` | Boolean
```

Comment:
We should still allow `AllowRBS` and deprecate it instead

Reply:
Sure, is there a common way of deprecating rubycop config values?
Or just "print deprecation warning that redirects to new config when set"?